src/components/ActionMenu/MenuItems/AddNew.js

```diff
@@ -0,0 +1,41 @@
+import React, {
+  memo,
+} from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { Icon, Button } from '@folio/stripes/components';
+
+import sharedCss from '../../../shared.css';
+import { createLayerURL } from '../../../utils';
+import { LAYER_TYPES } from '../../../utils/constants';
+
+export const AddNew = memo(props => {
+  const {
+    caption,
+    menu,
+    location,
+  } = props;
+
+  return (
+    <Button
```

Comment:
It seems the button (and its internal content - Icon and etc) can be moved to the new base item component and just be customized by different values of properties. For now, it looks like a code duplication in the menu items (AddNew, Default, ExportSelected)